x-pack/plugins/screenshotting/server/browsers/download/index.ts

```diff
@@ -29,57 +29,49 @@ export async function download(paths: ChromiumArchivePaths, logger?: Logger) {
   removedFiles.forEach((path) => logger?.warn(`Deleting unexpected file ${path}`));
 
   const invalidChecksums: string[] = [];
-  await Promise.all(
-    paths.packages.map(async (path) => {
-      const { archiveFilename, archiveChecksum } = path;
-      if (!archiveFilename || !archiveChecksum) {
-        return;
-      }
 
-      const resolvedPath = paths.resolvePath(path);
-      const pathExists = existsSync(resolvedPath);
+  const { archiveFilename, archiveChecksum } = pkg;
```

Comment:
Removed the loop that downloads every browser. This `download` function is already called multiple times when creating the multiple distributables for each platform/architecture.